Link edward statically to avoid having to package all CHICKEN
Scheme eggs on which edward depends. Long term, it would be
nice to package CHICKEN eggs in aports.

diff -upr edward-0.4.0.orig/edward.egg edward-0.4.0/edward.egg
--- edward-0.4.0.orig/edward.egg	2022-07-28 20:35:20.180509189 +0200
+++ edward-0.4.0/edward.egg	2022-07-28 20:35:29.350522791 +0200
@@ -5,7 +5,6 @@
  (dependencies r7rs srfi-1 srfi-14 srfi-37 matchable posix-regex)
  (test-dependencies test)
  (component-options
-   (linkage dynamic) ;; don't build static to reduce compile time
    (csc-options "-d3" "-O3" "-X" "r7rs" "-R" "r7rs"))
  (components
    (extension edward.util
@@ -75,7 +74,7 @@
    (program bin/edward
             ;; Uncomment to build a fully static version of edward.
             ;; Also requires removing (linkage dynamic) in component-options above.
-            ;(linkage static)
+            (linkage static)
             ;(link-options "-L" "-static")
 
             (install-name "edward")
